File: CICS 160 assignment 4/Employee.java
public class Employee extends Person {
    private String department;
    public Employee(String newName, String address, String phone, String department){
        super(newName,address,phone);
        this.department = department;

    }

    void setDepartment(String department){
        this.department = department;

    }

    String getDepartment(){
        return this.department;
    }

    public String toString(){
        return (super.toString()+"Department ="+getDepartment()) +("\n");


    }

public static void main (String[] args){
    System.out.println("testing Employee");
    Employee aEmployeeObject = new Employee("Jaime", "Amherst", "413-545-8929","IT");
    System.out.println(aEmployeeObject.toString());
    }
}

File: assignment 3 CICS160/Employee.py
from Person import Person 
class Employee(Person):
    def __init__(self, newName="none", address="none", phone="999-999-9999", department="not assigned"):
        super().__init__(newName,address,phone)

        self.department = department


    def __str__(self):
         return super().__str__()+ "Department  :"+ self.department+'\n'
    
    def getDepartment(self):
        return(self.department)

    def setDepartment(self,departmentname):
        self.department = departmentname

